src/test/java/htsjdk/tribble/bed/BEDCodecTest.java

```diff
@@ -226,4 +226,15 @@ private void createIndex(File testFile, File idxFile) throws IOException {
     public void testGetTabixFormat() {
         Assert.assertEquals(new BEDCodec().getTabixFormat(), TabixFormat.BED);
     }
+
```

Comment:
I think that I don't need to implement a new test for this. A `FeatureReader` for bgzip compressed files and tabix index queries is implemented in `FeatureReaderTest.testBedQuery()`.